tests/group/group_async_work_group_copy.cpp

```diff
@@ -89,38 +90,31 @@ class TEST_NAME : public util::test_base {
         "sycl::cl_ulong");
     for_type_and_vectors<check_type, sycl::cl_float>(log,
         "sycl::cl_float");
+#endif
 
 #ifdef INT8_MAX
-    if (!std::is_same<sycl::cl_char, std::int8_t>::value)
```

Comment:
I wonder how these tests could have been false at the first place.

Reply:
I don't think they can, integer widths are part of the spec.